.gitattributes

```diff
@@ -1,3 +0,0 @@
-*.sh eol=lf
```

Comment:
the original repo still has these files, why are they removed here?

I compare with https://github.com/ultimate-comparisons/ultimate-comparison-BASE/tree/update, is this right?